### PR TITLE
Add support for GPT BIOS images

### PR DESCRIPTION
**Pull request by mdavis777**

### CODE REVIEW

File: meta-mender-core/classes/mender-setup-bios.inc [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Make sure that GRUB is flashed to first non-partition table sector.
 MENDER_IMAGE_BOOTLOADER_FILE_DEFAULT_mender-grub_mender-bios = "grub-core.img"
-MENDER_IMAGE_BOOTLOADER_BOOTSECTOR_OFFSET_DEFAULT_mender-grub_mender-bios = "1"
+MENDER_IMAGE_BOOTLOADER_BOOTSECTOR_OFFSET_DEFAULT_mender-grub_mender-bios_mender-image-bios = "1"
+MENDER_IMAGE_BOOTLOADER_BOOTSECTOR_OFFSET_DEFAULT_mender-grub_mender-bios_mender-image-gpt = "34"
```
Comment (kacf) [inline meta-mender-core/classes/mender-setup-bios.inc]: Is this number 34 documented anywhere? Might be good to have a comment reference to that.

**Comment by mdavis777** [comment]
@kacf 
Both issues fixed.

**Comment by kacf** [comment]
Thanks! I have opened #634 for continued testing.

**Comment by kacf** [comment]
Alright, merged!

**Pull request merged**